KakapoExample/KakapoTests/KakapoServerTests.swift

```diff
@@ -107,6 +107,83 @@ class KakapoServerTests: QuickSpec {
                 expect(responseURL?.absoluteString).toEventually(beNil())
                 expect(responseError).toNotEventually(beNil())
             }
+            
+            it("should not call the handler when requesting a registered url but passing different HTTPMethod") {
+                var info: URLInfo? = nil
+                var responseURL: NSURL? = NSURL(string: "")
+                var responseError: NSError? = NSError(domain: "", code: 1, userInfo: nil)
+                
+                KakapoServer.del("/users/:id") { request in
+                    // Shouldn't reach here
+                    info = request.info
+                }
+                
+                let request = NSMutableURLRequest(URL: NSURL(string: "/users/1")!)
+                request.HTTPMethod = "PUT"
+                NSURLSession.sharedSession().dataTaskWithRequest(request) { (_, response, error) in
+                    // Response will be nil since error
+                    responseURL = response?.URL
+                    responseError = error
+                    }.resume()
+                
+                expect(info?.params).toEventually(beNil())
```

Comment:
just info will be nil, maybe also an `XCTFail()`(or quick equivalent) in the closure